### PR TITLE
Master PR: Fix script for create_transifex_resource.sh and pre_translate.sh

### PR DESCRIPTION
**Pull request by adissadis**

### CODE REVIEW

**Comment by adissadis** [comment]
This PR to merge what i did in branch develop to master branch

cc @timlinux

File: scripts/pre_translate.sh [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-INASAFE_DEV_PATH=$HOME/dev/python/inasafe-dev/
+INASAFE_DEV_PATH=$HOME/dev/python/inasafe/
```
Comment (timlinux) [inline scripts/pre_translate.sh]: I don't think we need any of these python setup lines anymore...or QGIS_PREFIX_PATH...

**Comment by timlinux** [comment]
Hi @adissadis 

Well done for wading through all the details to get the build process cleaned up and remove redundant files. I made a small note about removing python and QGIS path setup stuff in the scripts - but we can do that in a follow up commit.

LGTM

File: scripts/pre_translate.sh (hunk 1 above)
Comment (adissadis) [inline scripts/pre_translate.sh]: @timlinux so did you mean we give # in those line?

**Pull request merged**